pubsub/tests/system.py

```diff
@@ -14,6 +14,7 @@
 
 from __future__ import absolute_import
 
+from datetime import datetime
```

Comment:
Please import the module, not the type (it's a not-so-fun collision, but `datetime.datetime` is unambiguous)